app/controllers/pieces_controller.rb

```diff
@@ -0,0 +1,65 @@
+class PiecesController < ApplicationController
+  before_action :set_piece, only: %i[show edit update destroy]
+
+  # GET /pieces/1
+  # GET /pieces/1.json
+  def show
+    @current_piece = Piece.find(params[:id])
+    @game = @current_piece.game
+  end
```

Comment:
@mscoutermarsh Before I added @game here, the view kept crashing. I put a pry in and found out that @game was nil. Not sure if this is the best way to go about this, but it works now.

Reply:
Looks good!